src/features/Navigation/LeftContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
```

Comment:
LeftContainer or LogoContainer? 
Its on the left-side of the navigation-bar, so it kind of makes sense to be LeftContainer, but maybe its more readable if it is LogoContainer?

Reply:
Yes, I think Logo container is a much more descriptive name.